### ISSUE TITLE
SegFault on 0x7ffff5cca700

### ISSUE THREAD

**Issue by olivierb2** [coturn/coturn#896]
I installed a new Ubuntu 20.04 server last week and I realize that my turn server is crashing when I'm doing test with [WebRTC samples](https://webrtc.github.io/samples/).

For information, I already installed several coturn with on same OS and same hosting provider that not suffering of this issue.

I tried to debug with gdb, but I'm not quite confortable with this tool.

What I've done so far without any success.

- Uninstall the package, purge configuration and reinstalling (from universe 4.5.1.1-1.1ubuntu0.20.04.2).
- Compiled last version from github (debian tag 4.5.2-3).

I attempt to get more debug info, but I only got this from gdb

~~~
Thread 2 "coturn" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7ffff5cca700 (LWP 7774)]
0x00005555555909ad in turn_mutex_lock (mutex=mutex@entry=0x60010) at src/apps/common/ns_turn_utils.c:71
71	src/apps/common/ns_turn_utils.c: No such file or directory.
(gdb) next
Couldn't get registers: No such process.
Couldn't get registers: No such process.
(gdb) [Thread 0x7fffeeffd700 (LWP 7779) exited]
[Thread 0x7fffef7fe700 (LWP 7778) exited]
[Thread 0x7fffeffff700 (LWP 7777) exited]
[Thread 0x7ffff4ac7700 (LWP 7776) exited]
[Thread 0x7ffff52c8700 (LWP 7775) exited]
[Thread 0x7ffff5cca700 (LWP 7774) exited]
warning: Error removing breakpoint 0
warning: Error removing breakpoint 0
warning: Error removing breakpoint 0
warning: Error removing breakpoint 0
warning: Error removing breakpoint 0
warning: Error removing breakpoint 0

Program terminated with signal SIGSEGV, Segmentation fault.
The program no longer exists.
~~~

I'm even wondering now if the issue is not coming from lower issue as my hosting provider is providing me VPS machine.

**Comment by ggarber** [comment]
Did you find the root cause?  What is the difference between those other coturn servers you had installed and the one that is crashing?  Only the ubuntu version number?   Is there any chance you can use the existing docker images instances instead of building your own?

**Comment by ggarber** [comment]
closing for now.  Please provide more info and/or retry with the latest version released (4.6.0) that includes some fixes for crashes and reopen this ticket if needed.
